recipes/opensim/meta.yaml

```diff
@@ -0,0 +1,93 @@
+{% set version = "3.3" %}
+
+package:
+    name: opensim
+    version: {{ version }}
+
+source:
+    fn: OpenSim-{{ version }}-source.zip
+    url: https://simtk.org/frs/download_start.php/file/5025/OpenSim-{{ version }}-source.zip
+    sha256: 597482e78202f8480d205f10084ca4663236d76aa59de00a0408cdefbac43e7c
+
+build:
+    number: 200
+    # NOTE : For C++11 builds, vc14/VS 2015 are the only acceptable targets
+    # which only Python 3.5 supports. So skip previous Python versions.
+    skip: true                # [win and not py35]
```

Comment:
conda-forge supports 3.6 and python packages are being updated atm.
py35 and py36 uses the same MSVC version, so you don't need to compile against different vc versions.
If you are interested, you can get rid of python requirement since you need only one vc version. (If you need 2 or more, you need python as conda-forge support multiple builds based on python version and numpy version only)
Here's an example, https://github.com/conda-forge/mpfr-feedstock/blob/master/recipe/meta.yaml